backend/django/create-model-and-api.mo.md

```diff
@@ -0,0 +1,219 @@
+# [MO] Create a model and an api route (~20 mins)
+
+## Owner: Alice Breton
+
+## Prerequisites (~2 mins)
+
+- Have a django project installed, [see this](https://github.com/bamlab/dev-standards/blob/master/backend/django/getting-started.mo.md).
+- Have an app (in this example Publications).
+- In this example we will use an already existing model 'city' in a 'locations' app, to show how to integrate other models with foreign keys.
```

Comment:
Above you specify "in this example Publications" and just after "'city' in a 'locations' app". Which one to choose ?